src/so_vits_svc_fork/utils.py

```diff
@@ -256,20 +260,20 @@ def load_checkpoint(
             warnings.simplefilter("ignore")
             safe_load(optimizer, checkpoint_dict["optimizer"])
 
-    LOG.info(f"Loaded checkpoint '{checkpoint_path}' (iteration {iteration})")
-    return model, optimizer, learning_rate, iteration
+    LOG.info(f"Loaded checkpoint '{checkpoint_path}' (epoch {epoch})")
+    return model, optimizer, learning_rate, epoch
 
 
 def save_checkpoint(
     model: torch.nn.Module,
     optimizer: torch.optim.Optimizer,
     learning_rate: float,
-    iteration: int,
+    epoch: int,
```

Comment:
I am concerned about destructive changes, so please just change this variable names back to the original.

Reply:
I left the variable name as `iteration` in torch checkpoint structure to avoid any breaking changes, so not sure what else it can destruct.

But no problem, I updated my PR, and now it updates just log message.